bundles/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoHubReadDcbResponse.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.neohub.internal;
+
+import javax.measure.Unit;
+
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.unit.ImperialUnits;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * A wrapper around the JSON response to the JSON READ_DCB 100 request
+ *
+ * @author Andrew Fiddian-Green - Initial contribution
+ * 
+ */
+public class NeoHubReadDcbResponse {
+
+    private static final Gson GSON = new Gson();
+
+    @SerializedName("CORF")
+    private String degreesCorF;
+    @SerializedName("Firmware version")
+    private Integer firmwareVersion;
+
+    public Unit<?> getTemperatureUnit() {
+        return "F".equalsIgnoreCase(degreesCorF) ? ImperialUnits.FAHRENHEIT : SIUnits.CELSIUS;
+    }
+
+    public String getFirmwareVersion() {
+        return firmwareVersion.toString();
```

Comment:
Please handle the case if `firmwareVersion` is null

Reply:
@cpmeister thanks for pointing this out; in fact firmwareVersion is never used code, so I think I will remove it entirely; I will probably remove other elements of never used code at the same time.

Reply:
Done